src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -301,7 +317,8 @@ public void TimeOut_SetThenGet_ValuesMatch(Uri remoteServer)
                 var sw = Stopwatch.StartNew();
                 WebException exception = Assert.Throws<WebException>(() =>
                 {
-                    var response = (HttpWebResponse)request.GetResponse();
+                    var response = request.GetResponse();
+                    response.Dispose();
```

Comment:
Except now with your change you don't know whether the exception came from GetResponse or Dispose.  This should be reverted.